packages/gui/src/hooks/useNFTHash.ts

```diff
@@ -1,9 +1,45 @@
-import type NFTInfo from '@chia/api';
-import useVerifyURIHash from './useVerifyURIHash';
+import mime from 'mime-types';
```

Comment:
Use `npx lerna add mime-types@2.1.35 --scope @chia/gui` to add to `packages/gui/package.json`

Reply:
Done.